.github/workflows/helm.yaml

```diff
@@ -0,0 +1,74 @@
+name: Helm
+on:
+  push:
+    paths:
+    - '.github/workflows/helm.yaml'
+    - 'charts/**'
+  pull_request:
+    paths:
+    - '.github/workflows/helm.yaml'
+    - 'charts/**'
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Kubectl
+      uses: azure/setup-kubectl@v1
+    - name: Setup Helm
+      uses: azure/setup-helm@v1
+      with:
+        version: '3.6.3'
+    - name: Lint Chart
+      working-directory: ./charts/airbyte
+      run: ./ci.sh lint
+
+  generate-docs:
+    name: Generate Docs Parameters
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+    - uses: actions/checkout@v2
+    - name: Checkout bitnami-labs/readme-generator-for-helm
+      uses: actions/checkout@v2
+      with:
+        repository: 'bitnami-labs/readme-generator-for-helm'
+        ref: '55cab5dd2191c4ffa7245cfefa428d4d9bb12730'
+        path: readme-generator-for-helm
+    - name: Install readme-generator-for-helm dependencies
+      working-directory: readme-generator-for-helm  
+      run: npm install -g
+    - name: Test can update README with generated parameters
+      working-directory: charts/airbyte
+      run: ./ci.sh check-docs-updated
+
+  install:
+    name: Install
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Kubectl
+      uses: azure/setup-kubectl@v1
+    - name: Setup Helm
+      uses: azure/setup-helm@v1
+      with:
+        version: '3.6.3'
+    - name: Setup Kind Cluster
+      uses: helm/kind-action@v1.2.0
+      with:
+        version: "v0.11.1"
+        image: "kindest/node:v1.21.1"
+    - name: Install airbyte chart
+      working-directory: ./charts/airbyte
+      run: ./ci.sh install
+    - if: always()
+      name: Print diagnostics
+      working-directory: ./charts/airbyte
+      run: ./ci.sh diagnostics
+    - if: success()
+      name: Test airbyte chart
+      working-directory: ./charts/airbyte
+      run: ./ci.sh test
```

Comment:
@davinchia, This job in the workflow:

1. Spins up a kind cluster
2. Does a helm install of the chart with the default `values.yaml` file and waits(with the `--wait 600s` for all the pods to be ready according to the readiness probe. If there is a misconfig in the env-configmap, the pod should crash loop and the install will fail. I've seen this locally when it can't reach postgres or mino it just crash loops.
3. It then runs a `helm test` which spins up a test pod to make sure the web frontend `index.html` is able to be fetched. (If there are other good tests, like hitting specific routes on the server service, those can be added as well for a more comprehensive test)

You can find the an example and output of the action here(https://github.com/jonstacks/airbyte/runs/3613634997)